src/assets/README.md

```diff
@@ -7,31 +7,31 @@ The ILDH uses [Laravel Mix](https://laravel-mix.com) as a wrapper for webpack to
 
```

Comment:
This needs a significant update, as it still refers to Laravel Mix; probably best to replace most of it with https://github.com/fluid-project/trivet/blob/main/src/assets/README.md.